front/components/agent_builder/capabilities/knowledge/utils.ts

```diff
@@ -153,9 +152,7 @@ export function getKnowledgeDefaultValues({
       );
     }
 
-    return mcpServerViews.find(
-      (view) => view.server.name === SEARCH_SERVER_NAME
-    );
+    return null;
```

Comment:
Can we add a comment that if it's new we set null as a default value, and will set it once they land in the processing method section based on data source? (that's my understanding but is it correct?)

Reply:
It is correct 👍🏼